src/networking/Server.ts

```diff
@@ -22,11 +22,11 @@ declare var Util;
 // works with Node.js modules. http://stackoverflow.com/questions/13444064/typescript-conditional-module-import-export
 try
 {
-    var Events = require('./Events');
-    var ServerUtilies = require('./ServerUtilies');
-    var GameLobby = require('./GameLobby');
-    var ServerSettings = require('./ServerSettings');
-    var Lobby = require('./Lobby');
+    //var Events = require('./Events');
+    //var ServerUtilies = require('./ServerUtilies');
+    //var GameLobby = require('./GameLobby');
+    //var ServerSettings = require('./ServerSettings');
+    //var Lobby = require('./Lobby');
```

Comment:
?

Reply:
Visual studio was giving the error "Duplicate name" for those guys